Jenkinsfile

```diff
@@ -26,7 +26,7 @@ pipeline {
     stage('Build') {
       steps {
         echo 'Building Docker Image'
-        sh 'docker build -t hapi-fhir-jpaserver-starter .'
+        sh 'docker build -t hapi-fhir-jpaserver-starter:latest .'
```

Comment:
Adding latest here should not be needed. If you refer to other examples that we have in other repos, we don't do this.